### DISCUSSION TITLE
Please update doc!

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#14447] [General - Components / Directives / etc] [answered]
https://quasar.dev/quasar-cli/commands-list#create 

It suggest to use quasar create, but it is deprecated

**Comment by rstoenescu** [comment, marked as answer]
Docs are correct. Please note the highlight below.

<img width="565" alt="image" src="https://user-images.githubusercontent.com/4665641/191260620-2db88ea8-1ad5-4147-b06c-b5f9dd208a8d.png">
